standard_template/standard/clause_specification_text.adoc

```diff
@@ -338,24 +338,23 @@ Example of a vertical CRS, here representing height above the NAV88 datum:
 
 ### 5.2. Temporal Reference Systems
 
-Time is referenced by a temporal reference system (temporal RS).
-In this specification, only a string-based notation for time values is defined.
+Time is referenced by a temporal reference system (temporal RS). In the current specification, only a string-based notation for time values is defined. Future versions of this specification may allow for alternative notations, such as recording time values as numeric offsets from a given temporal datum (e.g. "days since 1970-01-01").
```

Comment:
i don't object to this extra sentence, but i'm a little surprised to see a 'future may' included in this section of the specification.
if it's felt to be of value, then please disregard this comment, but i'd slightly stick to the original and leave this as implied, rather than state a conditional future.

Reply:
You're right, and I wondered that too. I added that sentence as it's frequently asked by users, but it could be that the sentence doesn't really belong in a specification, but in some kind of FAQ or guide. Maybe we can discuss on the next call.

Reply:
Iagree to leave in the infromative text, as itmakes it easier to read. the text style is really closer to W3C documents rather than ISO